appveyor.yml

```diff
@@ -6,9 +6,9 @@ clone_depth: 3
 before_build:
   - ps: $env:Date="$(git show -s --date=short --format='%ad')"
   - appveyor DownloadFile http://releases.llvm.org/5.0.0/llvm-5.0.0.src.tar.xz
```

Comment:
You didn't change the two 5.0.0 here to 6.0

Reply:
Pull request #2 - Create README.md

Reply:
sorry my mistake when changing code

Reply:
![esboco 18](https://user-images.githubusercontent.com/38867013/39454022-4cf16c2e-4caf-11e8-8639-53f71fca846f.png)